ketcher-autotests/tests/File-Management/Open-And-Save-Files/Open-File/Load-Fragment/load-fragment.spec.ts

```diff
@@ -0,0 +1,243 @@
+import { test, expect, Page } from '@playwright/test';
+import {
+  AtomButton,
+  clickInTheMiddleOfTheScreen,
+  pressButton,
+  takeEditorScreenshot,
+  waitForLoad,
+  openFileAndAddToCanvas,
+  openPasteFromClipboard,
+  waitForPageInit,
+  selectAtomInToolbar,
+  selectLeftPanelButton,
+  dragMouseTo,
+  LeftPanelButton,
+  getCoordinatesOfTheMiddleOfTheScreen,
+  moveMouseToTheMiddleOfTheScreen,
+  resetCurrentTool,
+  receiveFileComparisonData,
+  saveToFile,
+  moveOnAtom,
+} from '@utils';
+import { getKet } from '@utils/formats';
+
+async function moveElement(
+  page: Page,
+  atomLabel: string,
+  atomNumber: number,
+  xShiftForElement = 350,
+  yShiftForElement = 150,
+) {
+  const { x, y } = await getCoordinatesOfTheMiddleOfTheScreen(page);
+  const pointXToMoveElement = x - xShiftForElement;
+  const pointYToMoveElement = y - yShiftForElement;
+
+  await page.getByTestId('hand').click();
+  await moveOnAtom(page, atomLabel, atomNumber);
+  await dragMouseTo(pointXToMoveElement, pointYToMoveElement, page);
+}
+
+test.describe('load as fragment (Add to Canvas) srtuctures from files with different formats', () => {
+  test.beforeEach(async ({ page }) => {
+    await waitForPageInit(page);
+  });
+
+  test.afterEach(async ({ page }) => {
+    await takeEditorScreenshot(page);
+  });
+
+  test('Load Fragment - Load as fragment (Add to Canvas) - mol-files', async ({
```

Comment:
You can simplify a bunch of test cases just by using `for` loop:
```
const testCasesForOpeningFiles = [{
  firstFile: 'Molfiles-V2000/glutamine.mol',
  secondFile: 'Molfiles-V2000/cyclopentyl.mol',
}, 
// add other files
 {
}];

for (const testCase of testCasesForOpeningFiles) {
   const fileExtension = testCase.firstFile.split('.').at(-1)
   test(`Load Fragment - Load as fragment (Add to Canvas) - ${fileExtension}-files`, async ({
    page,
  }) => {
    /*
     * Test case: EPMLSOPKET-1860, EPMLSOPKET-1854, EPMLSOPKET-1861, EPMLSOPKET-2933
     */
    await openFileAndAddToCanvas(firstFile, page);
    // you can create structures, where there always 'O' is present
    // or you can add additional parameter for testCase, but, I guess, it would be simpler
    await moveElement(page, 'O', 0);
    await openFileAndAddToCanvas(secondFile, page);
  });
}
```

Reply:
Done